### PR TITLE
adding profile view

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/S5v6zkbI/6364-create-data-warehouse-views-8

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Create view for the profile table](https://trello.com/c/yLkT37HQ/6584-create-view-for-the-profile-table)

File: orcid-persistence/src/main/resources/db/updates/create_profile_view.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="CREATE-PROFILE-VIEW" author="Daniel Palafox">
+        <createView viewName="profile_view">
```
Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/create_profile_view.xml]: @DanielPalafox view name should be dw_profile

File: orcid-persistence/src/main/resources/db/updates/create_profile_view.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+            select orcid, date_created, last_modified, account_non_locked, profile_deactivation_date,
+            enable_developer_tools, last_login, using_2fa
+            from profile p
+            order by date_created
+        </createView>
+    </changeSet>
+
```
Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/create_profile_view.xml]: @DanielPalafox please add another changeset to grant read access to user `dw_user`

Comment (DanielPalafox) [reply orcid-persistence/src/main/resources/db/updates/create_profile_view.xml]: @amontenegro done!

File: orcid-persistence/src/main/resources/db/updates/create_profile_view.xml (hunk 1 above)
Comment (DanielPalafox) [reply orcid-persistence/src/main/resources/db/updates/create_profile_view.xml]: @amontenegro done!

**Pull request merged**